apps/docs/integrations/aws-ses.mdx

```diff
@@ -12,15 +12,15 @@ Get the [@react-email/render](https://www.npmjs.com/package/@react-email/render)
 <CodeGroup>
 
 ```sh npm
-npm install @react-email/render @aws-sdk/client-ses
+npm install @react-email/render @aws-sdk/client-ses @react-email/components
```

Comment:
Since `@react-email/components` also re-exports `@react-email/render`, can you change all of our integration guides to use only the `@react-email/components` so that they are cleaner? Having them both installed might get a bit confusing for new people coming around.

Reply:
Right, on it.

Reply:
I have a misunderstanding, since you said ```@react-email/components``` also re-exports ```@react-email/render``` here's my question :
- Doesn't it make this [page](https://react.email/docs/utilities/render) obsolete ? Here in the page there's an import for ```@react-email/components``` and the step 1 says install, ```@react-email/render``` when step 2 uses ```@react-email/components``` ? Or is it just a page to describe the utility package ?

Reply:
> Or is it just a page to describe the utility package ?

It's meant to be a page to describe the utility package and where it comes from. There it's fine the way it is because the code block importing `@react-email/components` is just an example of what you might have as an email template.

> Step 3 of each integration imports the @react-email/render package, won't that print an error if someone tries the code as it is ?

Yeah, so the idea is to also change these to import from `@react-email/components` instead of `@react-email/render`.

Reply:
Ok, I get it now, thanks for your answer. I made the adjustments.